README.md

```diff
@@ -17,95 +18,110 @@ This module adds a new exec provider capable of executing PowerShell commands.
-* **unless** - The `exec` will run unless the command returns 0.
+#####`command`
+Specifies the actual PowerShell command to execute. Must either be fully qualified or a search path for the command must be provided. Valid options: String. Default: Undefined.
 
-##Limitations
+#####`environment`
+Sets additional environment variables to set for a command. Valid options: String, or an array of multiple options. Default: Undefined.
 
- * This module requires PowerShell to be installed and the `powershell.exe` to be available in the system PATH.
- * Only supported on Windows Server 2003 and above, and Windows 7 and 8
+#####`logoutput`
+Defines whether to log command output in addition to logging the exit code. If you specify 'on_failure', it only logs the output when the command has an exit code that does not match any value specified by the `returns` attribute. Valid options: 'true', 'false', and 'on_failure'. Default: 'on_failure'.
 
-##Development
+#####`onlyif`
+Runs the exec only if the command returns 0. Valid options: String. Default: Undefined.
+
+#####`path`
+Specifies the search path used for command execution. Valid options: String of the path, or a ; separated list. Default: Undefined. 
+
+#####`refresh`
+Refreshes the command. Valid options: String. Default: Undefined.
+
+#####`refreshonly`
+Refreshes the command only when a dependent object is changed. Used with `subscribe` and `notify` [metaparameters](http://docs.puppetlabs.com/references/latest/metaparameter.html). Valid options: 'true', 'false'. Default: 'false'.
 
-Puppet Labs modules on the Puppet Forge are open projects, and community contributions are essential for keeping them great. We can’t access the huge number of platforms and myriad of hardware, software, and deployment configurations that Puppet is intended to serve.
+#####`returns`
+Lists the expected return code(s). An error will be returned if the executed command returns something else. Valid options: An array of acceptable return codes or a single value. Default: 0.
 
-We want to keep it as easy as possible to contribute changes so that our modules work in your environment. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+#####`timeout`
+Sets the maximum time in seconds that the command should take. Valid options: Number or string representation of a number. Default: 300.
+
+#####`tries`
+Determines the number of times execution of the command should be attempted. Valid options: Number or a string representation of a number. Default: '1'.
+ 
+#####`try_sleep`
+Specifies the time to sleep in seconds between `tries`. Valid options: Number or a string representation of a number. Default: Undefined.
+
+#####`unless`
+Runs the `exec`, unless the command returns 0. Valid options: String. Default: Undefined.
+
+##Limitations
+
+ * Only supported on Windows Server 2003 and above, and Windows 7 and 8.
```

Comment:
For now, I'll just change this sentence to end in "Windows 7 and above."  
If we have an official list of tested/working versions, it can be included in the future but for now I'd like to get these changes merged in.